src/System.Management.Automation/engine/SessionStateNavigation.cs

```diff
@@ -258,30 +256,25 @@ private string RemoveQualifier(string path, ProviderInfo provider, out string qu
             {
                 isProviderQualified = true;
 
-                int index = path.IndexOf("::", StringComparison.Ordinal);
-
-                if (index != -1)
-                {
-                    // remove the qualifier
-                    result = path.Substring(index + 2);
-                }
+                // remove the qualifier
```

Comment:
explain that qualified is separated by `::`

Reply:
Done.